src/services/web-monitoring-db.ts

```diff
@@ -72,6 +72,18 @@ export default class WebMonitoringDb {
             .then(data => parsePage(data.data));
     }
 
+    getPagesByDomains (domains: string[]): Promise<Page[]> {
+        const daysAgo = 3;
+        const dateEarlier = new Date(new Date().setDate(new Date().getDate() - daysAgo)).toISOString();
+        const fetches = domains.map(domain => {
+            return fetch(this.createUrl(`pages?site=${encodeURI(domain)}&capture_time=${dateEarlier}..`))
+                .then(response => response.json())
+                .then(data => data.data.map(parsePage));
+        });
+        return Promise.all(fetches)
+            .then(data => data.reduce((acc, arr) => acc.concat(arr), []));
```

Comment:
Good to know. Do you have some examples that I can test for? If it's too much time to get the data, can we mark as TODO?